src/WebResponseFactory.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Yiisoft\Yii\Web;
+
+use Psr\Http\Message\ResponseFactoryInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\StreamFactoryInterface;
+
+class WebResponseFactory implements WebResponseFactoryInterface
+{
+    protected StreamFactoryInterface $streamFactory;
+
+    protected ResponseFactoryInterface $responseFactory;
+
+    public function __construct(ResponseFactoryInterface $responseFactory, StreamFactoryInterface $streamFactory)
+    {
+        $this->streamFactory = $streamFactory;
+        $this->responseFactory = $responseFactory;
+    }
+
+    public function createResponse(int $code = 200, string $reasonPhrase = '', $data = null): ResponseInterface
```

Comment:
use constant Yiisoft\Http\Status::OK

Reply:
For what?

Reply:
For readability.

Reply:
It isn't necessary.

Reply:
But it better to be done.

Reply:
https://github.com/Nyholm/psr7/blob/master/src/Factory/Psr17Factory.php#L21